v2/internal/typescriptify/typescriptify.go

```diff
@@ -841,11 +841,10 @@ func (t *typeScriptClassBuilder) AddStructField(fieldName string, field reflect.
 	strippedFieldName := strings.ReplaceAll(fieldName, "?", "")
 	fqname := field.Type.Name()
 	classname := "null"
-
 	namespace := strings.Split(field.Type.String(), ".")[0]
-
+	fqname = t.prefix + field.Type.Name() + t.suffix
```

Comment:
I think that line 842 is unnecessary, right now:
```suggestion
	fqname := t.prefix + field.Type.Name() + t.suffix
```

Reply:
Exactly 🙂

Reply:
indeed, you are right @Goncalo-Marques , it is redundant, removed it